### PR TITLE
fix typo in faq.md

### PR DESCRIPTION
**Pull request by adriananin**
fixed typo in last line of document from rending to rendering

### CODE REVIEW

**Review by brophdawg11 — APPROVED** [review]
Thanks!

File: contributors.yml [hunk 1]
```diff
@@ -251,3 +251,4 @@
 - yracnet
 - yuleicul
 - zheng-chuang
+- adrian anin
```
Comment (brophdawg11) [inline contributors.yml:254]:
```suggestion
- adriananin
```

**Comment by brophdawg11** [comment]
Do you mind accepting the change to the CLA signature so it passes the bot check?  It needs to match your github username exactly

**Comment by timdorr** [comment]
Thanks!

**Pull request merged**